packages/styles/scss/components/pricing-table/_pricing-table.scss

```diff
@@ -152,6 +152,17 @@
       display: flex;
       justify-content: space-between;
     }
+
+    .#{$prefix}--pricing-table-cell-content {
+      display: flex;
+      align-items: start;
+      line-height: 1.25rem;
+
+      svg {
+        margin-inline-end: $spacing-03;
+        flex-shrink: 0;
+      }
+    }
```

Comment:
@m4olivei You know, it just dawned on me – should these styles be moved over to the Structured List stylesheet?

Reply:
Answered my own question. The wrapper element that has this class only appears in Pricing Table cells, which I believe means these styles are only necessary in this instance.

Reply:
Heh, fun timing. Was just coming here to say that. I think we could restructure that wrapper element to be part of the parent structured list component, but it gets tricky, b/c there are some cases for the general strucutred list where you really don't want to inherit the flexbox layout.